### DISCUSSION TITLE
Jamulus pb to open

### DISCUSSION THREAD

**Discussion by Mikovitch** [jamulussoftware/jamulus#2985] [Technical Support & Help]
Dear all, I'm trying to install Jamulus on my PC with Windows 10, the installation went well but nothing is opening when i tried to open it; any suggestion ?
thanks

**Comment by ann0see** [comment]
That’s strange. I could imagine that the ASIO driver hangs. Are you using ASIO4ALL?

**Comment by pljones** [comment]
Could you please list all the steps you followed, starting from which page you went to for the download.
